tests/test-constants.js

```diff
@@ -39,6 +39,8 @@ const DefaultMinGasPrice = 50;
 const DefaultMaxGas = 429496729;
 const DefaultTestSchemes = "BBS,BBSPlus,PS";
 
+const boolEnv = (value) => value === "true" || !!+value;
```

Comment:
Isn't this sufficient `const boolEnv = (value) => value === "true"`

Reply:
I guess it's expected that any non-zero number must have the same behavior as `true`.